### PR TITLE
fix #3076

### LINKED ISSUE #3076: Improve error message for "Error (log id: 312889): Error tokenizing data. C error: Expected 3 fields in line 10, saw 5"
In other works, make it more user friendly. Perhaps go with:
"Your file has more or less columns than headers; this is a common problem with Excel files. To fix, select the empty columns after your data and deleting them (even if they seem empty already) and do the same with the rows; they upload and try again."

### PR DESCRIPTION
**Pull request by antgonza**
Extra from fixing the above issue, this is also updating to the latests version of DbSchema the qdb/html files.

### CODE REVIEW

File: qiita_db/metadata_template/test/test_util.py [hunk 1]
```diff
@@ -265,6 +265,16 @@ def test_load_template_to_dataframe_with_nulls(self):
...
 
+    def test_load_template_to_dataframe_better_tokenizing_error_msg(self):
+        with self.assertRaises(RuntimeError) as error:
+            qdb.metadata_template.util.load_template_to_dataframe(
+                StringIO('sample_name\tcollection_timestamp\n'
+                         '2.Sample1\t2014-05-29 12:24:51\t\n'
+                         '2.Sample2\taaa\n'
+                         'xxx\tadfa\t\t\n'))
+        self.assertIn('Your file has more columns with values than headers.',
+                      str(error.exception))
```
Comment (ElDeveloper) [inline qiita_db/metadata_template/test/test_util.py:276]: Any reason not to use `assertRaisesRegex`? 

```suggestion
        with self.assertRaisesRegex(RuntimeError, 'Your file has more columns with values than headers'):
            qdb.metadata_template.util.load_template_to_dataframe(
                StringIO('sample_name\tcollection_timestamp\n'
                         '2.Sample1\t2014-05-29 12:24:51\t\n'
                         '2.Sample2\taaa\n'
                         'xxx\tadfa\t\t\n'))
```

File: qiita_db/metadata_template/util.py [hunk 2]
```diff
@@ -144,16 +144,26 @@ def load_template_to_dataframe(fn, index='sample_name'):
...
+            index_col=False,
+            comment='\t',
+            converters={index: lambda x: str(x).strip()})
+    except pd.errors.ParserError as e:
+        if 'tokenizing' in str(e):
+            msg = ('Your file has more columns with values than headers. To '
+                   'fix, make sure to delete any extra rows or columns; they '
+                   'might look empty because they have spaces. Then upload '
+                   'and try again.')
+            raise RuntimeError(msg)
```
Comment (ElDeveloper) [inline qiita_db/metadata_template/util.py:164]: Can you add this to the docstring?

**Review by ElDeveloper — CHANGES_REQUESTED** [review]
A few comments. One general question: what are the changes to the DB? Doesn't seem like this issue requires any DB updates, right?

**Comment by antgonza** [comment]
I'll address your suggestions soon. The changes are cause I downloaded the latest version of the DBSchema software and generated the html with the latest version so it's clearer; in other words, nothing really changed just the files to the latest version.

**Comment by ElDeveloper** [comment]
Makes sense, thanks!

**Comment by ElDeveloper** [comment]
Thanks @antgonza

**Pull request merged**